content/teaching-curriculum/quick-familiarity-4/_index.md

```diff
@@ -2,29 +2,35 @@
-- none
+  - markdown
 prerequisites:
   hard:
-  - teaching-curriculum/introducing-teach-like-a-champion
+    - teaching-curriculum/introducing-teach-like-a-champion
+protect_main_branch: false
 ready: true
-submission_type: link
+submission_type: repo
 tags:
-- teaching
+  - teaching
 title: Quick Familiarity - Part 4
 ---
 
-As mentioned in the introductory topic of the course, the point of the quick familiarity projects is to do a quick skim of all of the techniques in the book, just to introduce you to the ideas and make sure that we can all start using the common language that the book provides for talking about nuanced teaching practices. We’ll practice the techniques more thoroughly later, but for now let's get just acquainted with them.
+As mentioned in the introductory topic of the course, the point of the quick familiarity projects is to do a quick skim of all of the techniques in the book, just to introduce you to the ideas and make sure that we can all start using the common language that the book provides for talking about nuanced teaching practices. We’ll practice the techniques more thoroughly later, but for now, let's just get acquainted with them.
 
-[This 12 page book summary](https://drive.google.com/file/d/1ace5039zhdNbrd4CBgXz3GikFpPwMLru/view?usp=share_link) provides a really nice 1 page intro to the methodology that author Doug Lemov and team used to distil the best teaching practices from the best teachers in the country. 
+[This 12-page book summary](https://drive.google.com/file/d/1ace5039zhdNbrd4CBgXz3GikFpPwMLru/view?usp=share_link) provides a nice 1-page intro to the methodology that author Doug Lemov and team used to distill the best teaching practices from the best teachers in the country.
 
-To keep the projects bite-sized, we'll look at just 3 chapters at a time and do a little bit of light reflection and planning. 
+To keep the projects bite-sized, we'll look at just 3 chapters at a time and do some light reflection and planning.
 
-## Your Mission Objectives
-* Read through the summary of **Chapters 10, 11 and 12**. At first glance these chapters might not seem relevant to teaching coders, but actually some of the techniques could be super useful!!! 
-* For each of these 3 chapters: 
-    * (A) Describe which of the techniques you already apply pretty well, in general, and where and how it is that you tend do so. Also explain how it is that you know it works. 
-    * (B) Describe which of the techniques, if applied better or more consistently, you think has the greatest potential to improve your teaching practice. Be sure to motivate your choice with some sound reasoning.  
+## Your Mission Objectives:
 
-Describe each of the above in no more than 300 words. 
-Do this on a Google doc and submit a link to the doc. 
-MAKE SURE THAT THE SHARING PERMISSIONS ALLOW ANYONE ON THE INTERNET TO VIEW THE FILE!
\ No newline at end of file
+- Read through the first 5 pages of the summary pdf (up to the end of chapter 3).
```

Comment:
- The chapters here should be Chapters 10, 11 and 12
- Also format the headings consistently for all parts of the project. E.g. bold all the chapters that need to be covered